django/db/models/functions/json.py

```diff
@@ -0,0 +1,218 @@
+from django.db import NotSupportedError
+from django.db.models.constants import LOOKUP_SEP
+from django.db.models.expressions import Func, Value
+from django.db.models.fields.json import JSONField, compile_json_path
+from django.db.models.functions import Cast
+
+
+class JSONSet(Func):
```

Comment:
Let's create a separate commit for `JSONSet`+tests, and another commit for `JSONRemove`+tests. Then, create a separate commit for adding `KeyTransform.get_update_expression()` and the corresponding tests.

Reply:
Done :heavy_check_mark: